package.json

```diff
@@ -115,12 +115,11 @@
     "cli-table3": "^0.6.3",
     "dotenv": "16.3.1",
     "esbuild": "0.16.12",
-    "eslint": "^8.23.0",
-    "eslint-config-standard": "^11.0.0-beta.0",
+    "eslint": "^8.56.0",
+    "eslint-config-standard": "^17",
     "eslint-plugin-import": "^2.8.0",
     "eslint-plugin-mocha": "^10.1.0",
-    "eslint-plugin-n": "^15.7.0",
-    "eslint-plugin-node": "^5.2.1",
+    "eslint-plugin-n": "^16.6.2",
```

Comment:
This version doesn't support older versions of Node. Security fixes should be backportable to all release lines.

Reply:
Hmm... Do we run the linter with every supported version of Node?

Reply:
In some cases yes.